src/Maker/MakeForgottenPassword.php

```diff
@@ -0,0 +1,333 @@
+        return 'make:forgotten-password';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->setDescription('Creates a "forgotten password" mechanism')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeForgottenPassword.txt'))
+        ;
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        // initialize arguments & commands that are internal (i.e. meant only to be asked)
+        $command
+            ->addArgument('user-class')
+            ->addArgument('email-field')
+            ->addArgument('email-getter')
+            ->addArgument('password-setter')
+        ;
+
+        $interactiveSecurityHelper = new InteractiveSecurityHelper();
+
+        if (!$this->fileManager->fileExists($path = 'config/packages/security.yaml')) {
+            throw new RuntimeCommandException('The file "config/packages/security.yaml" does not exist. This command needs that file to accurately build the forgotten password form.');
+        }
+
+        $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
+        $securityData = $manipulator->getData();
+        $providersData = $securityData['security']['providers'] ?? [];
+
+        $input->setArgument(
+            'user-class',
+            $userClass = $interactiveSecurityHelper->guessUserClass(
+                $io,
+                $providersData,
+                'Enter the User class that should be used with the "forgotten password" feature (e.g. <fg=yellow>App\\Entity\\User</>)'
+            )
+        );
+        $io->text(sprintf('Implementing forgotten password for <info>%s</info>', $userClass));
+
+        $input->setArgument(
+            'email-field',
+            $interactiveSecurityHelper->guessEmailField($io, $userClass)
+        );
+        $input->setArgument(
+            'email-getter',
+            $interactiveSecurityHelper->guessEmailGetter($io, $userClass)
+        );
+        $input->setArgument(
+            'password-setter',
+            $interactiveSecurityHelper->guessPasswordSetter($io, $userClass)
+        );
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+        // This recipe depends upon Doctrine ORM, to save the token and update the user
+        ORMDependencyBuilder::buildDependencies($dependencies);
+
+        $dependencies->addClassDependency(
+            AbstractType::class,
+            'form'
+        );
+        $dependencies->addClassDependency(
+            Validation::class,
+            'validator'
+        );
+        $dependencies->addClassDependency(
+            TwigBundle::class,
+            'twig-bundle'
+        );
+        $dependencies->addClassDependency(
+            SecurityBundle::class,
+            'security'
+        );
+        $dependencies->addClassDependency(
+            SwiftmailerBundle::class,
```

Comment:
should be the new Symfony Mailer component instead of Swiftmailer.

I tried to install this bundle, but even though I installed mail, the dependency was swiftmailer, this error kept popping up